app/views/claims/qts_year.html.erb

```diff
@@ -12,8 +12,14 @@
         </span>
 
         <%= errors_tag current_claim, :qts_award_year %>
-        <%= form.select :qts_award_year, options_for_qts_award_year, { include_blank: true }, { class: css_classes_for_select(current_claim, :qts_award_year) } %>
-
+        <div class="govuk-radios">
```

Comment:
The `shared/error_summary` render call above will want `errored_field_id_overrides` set so that the error summary can correctly link the error message to the first radio in the list. See the [`mostly_teaching_eligible_subjects.html.erb`](https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/6e519a866117902775efee57a6245a53c152ec26/app/views/claims/mostly_teaching_eligible_subjects.html.erb#L3) partial for an example of how to do this